reframe/core/containers.py

```diff
@@ -19,11 +19,31 @@ class ContainerPlatform(abc.ABC):
     #: :default: :class:`None`
     image = fields.TypedField(str, type(None))
 
-    #: The commands to be executed within the container.
+    #: The command to be executed within the container.
     #:
-    #: :type: :class:`list[str]`
-    #: :default: ``[]``
-    commands = fields.TypedField(typ.List[str])
+    #: If no command is given, then the default command of the corresponding
+    #: container image is going to be executed.
+    #:
+    #: ..versionchanged:: 3.4.2
+    #:   Changed the attribute name from `commands` to `command` and its type
+    #:   to a string.
+    #:
+    #: :type: :class:`str` or :class:`None`
+    #: :default: :class:`None`
+    command = fields.TypedField(str, type(None))
```

Comment:
How about an `entrypoint` type to overwrite the `ENTRYPOINT` of the image?
Maybe by popular demand.

Reply:
In general you can put all the options you might need in the `options` field of the container platform. I am still thinking if we should have a separate `workdir` field.